### PR TITLE
Update Grafana dashboard link

### PR DESCRIPTION
**Pull request by deborahchua**
email-alert-api has now migrated to AWS.

### CODE REVIEW

File: source/manual/alerts/email-alert-api-app-healthcheck-not-ok.html.md [hunk 1]
```diff
@@ -186,6 +186,6 @@ decreased. If it's decreasing, then it means that emails are going out and
 there's probably a lot being processed. You can also check the
 [Email Alert API Metrics dashboard][dashboard].
 
-[dashboard]: https://grafana.publishing.service.gov.uk/dashboard/file/email_alert_api.json?refresh=10s&orgId=1
+[dashboard]: https://grafana.blue.staging.govuk.digita/dashboard/file/email_alert_api.json?refresh=10s&orgId=1
```
Comment (cbaines) [inline source/manual/alerts/email-alert-api-app-healthcheck-not-ok.html.md]: `grafana.staging.govuk.digital` should work as well (and it's shorter).

Additionally, you might be missing the *l* from `digital`.

Comment (sengi) [inline source/manual/alerts/email-alert-api-app-healthcheck-not-ok.html.md]:
```suggestion
[dashboard]: https://grafana.staging.govuk.digital/dashboard/file/email_alert_api.json
```

Comment (deborahchua) [reply source/manual/alerts/email-alert-api-app-healthcheck-not-ok.html.md]: Gah good spot thank you!!

File: source/manual/alerts/email-alert-api-app-healthcheck-not-ok.html.md [hunk 2]
```diff
@@ -186,6 +186,6 @@ decreased. If it's decreasing, then it means that emails are going out and
 there's probably a lot being processed. You can also check the
 [Email Alert API Metrics dashboard][dashboard].
 
-[dashboard]: https://grafana.publishing.service.gov.uk/dashboard/file/email_alert_api.json?refresh=10s&orgId=1
+[dashboard]: https://grafana.staging.govuk.digital//dashboard/file/email_alert_api.json?refresh=10s&orgId=1
```
Comment (cbaines) [inline source/manual/alerts/email-alert-api-app-healthcheck-not-ok.html.md]: You've got `//` here, which will probably still work, but only one `/` is necessary.

**Pull request merged**